pytest_robotframework/_internal/robot/utils.py

```diff
@@ -241,6 +243,7 @@ def cli_defaults(settings_class: Callable[[dict[str, object]], _BaseSettings]) -
     # the cwd gets updated, robot will still run with the outdated cwd.
     # we set it in this wacky way to make sure it never overrides user preferences
     _BaseSettings._cli_opts["OutputDir"] = ("outputdir", ".")  # pyright:ignore[reportUnknownMemberType,reportPrivateUsage]
+    RobotSettings._extra_cli_opts["ConsoleType"] = ("console", "quiet")  # pyright:ignore[reportUnknownMemberType,reportPrivateUsage,reportArgumentType]
```

Comment:
this will prevent it from printing the path to the robot log

Reply:
done